index.js

```diff
@@ -32,6 +39,22 @@ module.exports = {
   env: {
     browser: true,
     node: true,
+    // we are using optional chaining and nullish coalescing
+    es2020: true,
+  },
+  globals: {
+    // these globals help IDEs from displaying these as undefined
+    localStorage: true,
+    process: false,
+    Promise: true,
+  },
+  settings: {
+    // the src folder can be used as the resource root for imports
+    'import/resolver': {
+      node: {
+        paths: ['src'],
```

Comment:
>an array of absolute paths which will also be searched 
>think NODE_PATH 

https://www.npmjs.com/package/eslint-import-resolver-node

>NODE_PATH was originally created to support loading modules from varying paths before the current module resolution algorithm was defined.
>NODE_PATH is still supported, but is less necessary now that the Node.js ecosystem has settled on a convention for locating dependent modules. Sometimes deployments that rely on NODE_PATH show surprising behavior when people are unaware that NODE_PATH must be set. Sometimes a module's dependencies change, causing a different version (or even a different module) to be loaded as the NODE_PATH is searched.
>Additionally, Node.js will search.... These are mostly for historic reasons.
>It is strongly encouraged to place dependencies in the local node_modules folder. These will be loaded faster, and more reliably.

https://nodejs.org/api/modules.html#modules_loading_from_the_global_folders

`NODE_PATH` sounds like magic that causes surprises. What are the benefits of altering the import path? (to weigh against the surprises)

Reply:
This is so you can use src as a resource root.

Reply:
I use these `import/resolver` settings as well in my eslint setup. create-react-app lets you set `src` as a base url for path resolution through a `jsconfig.json` file. It then uses that setting to set the proper aliases and module resolution settings in webpack.config. If you don't also include the eslint setting here then eslint will complain about your imports, even if they're working correctly.

Reply:
Think it would be great to offer the same functionality in one app (this would need to be in a separate PR to one-app-bundler).

---
For reference, one-app-bundler's webpack config:
https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-bundler/webpack

Where CRA implements this:
https://github.com/facebook/create-react-app/pull/7755/files